database/database_test.go

```diff
@@ -29,11 +29,11 @@ func init() {
 		log.Fatal(err)
 	}
 
-	hep.SIP.CseqMethod = "INVITE"
-	hep.SIP.CallID = "te\"st\""
-	hep.SIP.UserAgent = "F\"box\"!"
-	hep.SIP.FromHost = "192.168.111.111\x00"
-	hep.CID = "te\"st\""
+	hep.ParsedSIP.CseqMethod = "INVITE"
```

Comment:
In Go short names are prefered. Why not let it just SIP?

Reply:
Context. Then it's evident that the SIP is the result of earlier parsing. I get that it will take eyes a bit to adjust to a longer variable name - but code readability is important, no? It all gets compiled and Go doesn't care how short or long it is. SIP can stay if it hurts your eyes :)

Reply:
Yes it really hurt my eyes badly ;)

Reply:
jfc XD